### PR TITLE
Add more test for jca integration test.

### PR DESCRIPTION
**Pull request by han-gao**
Modify the jca test cases.

### CODE REVIEW

**Review by yiliuTo — CHANGES_REQUESTED** [review]

**Comment by yiliuTo** [comment]
The part of logic in setEnvironmentProperty method  is used in multiple test class, how about move it to util class?

**Comment by yiliuTo** [comment]
Rename variable cf

File: sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultKeyStoreTest.java [hunk 1]
```diff
@@ -123,7 +124,14 @@ public void testEngineGetKey() {
 
     @Test
     public void testEngineAliases() {
-        assertEquals(keystore.engineAliases().nextElement(), certificateName);
+        Enumeration<String> allAliases = keystore.engineAliases();
+        boolean flag = false;
+        while (allAliases.hasMoreElements()) {
```
Comment (yiliuTo) [inline sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultKeyStoreTest.java:129]: how about
assertTrue(keystore.engineAliases().hasMoreElements());
assertEquals(keystore.engineAliases().nextElement(), certificateName);

Comment (han-gao) [reply sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultKeyStoreTest.java:129]: Thinks.

**Comment by chenrujun** [comment]
Hi, @han-gao , please investigate the pipeline failure.

**Comment by chenrujun** [comment]
Hi, @han-gao .
Please resolve the conflicts.

**Review by chenrujun — APPROVED** [review]
LGTM.

File: sdk/keyvault/azure-security-keyvault-jca/pom.xml [hunk 2]
```diff
@@ -191,6 +191,12 @@
           <version>3.6.28</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-core-test</artifactId>
+          <version>1.6.2</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+          <scope>test</scope>
+        </dependency>
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/pom.xml:199]: Where do we use it?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/keyvault/azure-security-keyvault-jca/pom.xml (hunk 2 above)
Comment (yiliuTo) [reply sdk/keyvault/azure-security-keyvault-jca/pom.xml:199]: This library is used to show our test info in the pipeline, you could check [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=915354&view=logs&j=bb4251f5-b8e9-5761-71ac-1cf2c72c9b27&t=231b82bd-ed1a-5f58-da3e-bace0d3d8d96&l=307), or there wouldn't be any logs that if our tests are scheduled or not. Same for the live test pipeline, do you think whether we should keep this dependency?

**Review by chenrujun — APPROVED** [review]
LGTM.

**Comment by chenrujun** [comment]
Hi, @han-gao , could you please fix the pipeline failure?

**Comment by chenrujun** [comment]
Hi, @han-gao , could you please fix the pipeline failure?

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**